coverage.out

```diff
@@ -0,0 +1 @@
+mode: count
```

Comment:
I think this needs to be excluded from the PR

Reply:
oups...didn't notice it. I have remove it. Thanks